src/adapter/src/catalog.rs

```diff
@@ -2209,7 +2212,7 @@ impl Source {
     /// Returns if this source is a webhook source, which indicates `environmentd` is responsible
     /// for receiving and pushing data to storage.
     pub fn is_webhook(&self) -> bool {
-        matches!(self.data_source, DataSourceDesc::Webhook)
+        matches!(self.data_source, DataSourceDesc::Webhook { .. })
     }
```

Comment:
Can we cover this method with a test case?

Reply:
This code wasn't used, so I deleted it :)